website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -341,3 +341,80 @@ In the configuration format for the model SQL file:
 </File>
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+
+## Dynamic Tables
+
+[Dynamic Tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-about) are Snowflake's flavor of Materialized Views.
+
+### Parameters
+
+dbt-snowflake requires the following parameters:
+
+- `TARGET_LAG`
+- `WAREHOUSE`
+
+To learn more about each parameter and what values it can take, see 
```

Comment:
We don't support `downstream` for `target_lag` yet. This will appear in the linked docs, so it should be called out here that we don't support it.